RInAppMessaging/Classes/InAppMessagingModule.swift

```diff
@@ -92,15 +92,17 @@ internal class InAppMessagingModule: AnalyticsBroadcaster,
         } else {
             diff = preferenceRepository.preference == nil ? [] : nil
         }
+        let isFirstUser = preferenceRepository.getUserIdentifiers().isEmpty && diff?.isEmpty == false
```

Comment:
Does this mean the first logged in user? i.e. non-default user

Reply:
Yes. I should probably use a better name for this var